<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>client</class>
 <widget class="QWidget" name="client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>654</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTableView" name="tableView"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="client_id">
         <property name="text">
          <string>client id</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="firstName">
         <property name="text">
          <string>firstName</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lastName">
         <property name="text">
          <string>lastName</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="phoneNumber">
         <property name="text">
          <string>Phone number</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="lineClient_id"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineFirstName"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineLastName"/>
       </item>
       <item>
        <widget class="QLineEdit" name="linePhoneNumber"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="add">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="delete_2">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
